### PR TITLE
Fix some broken bits of the NANDOs finder

### PR DESCRIPTION
**Pull request by sihugh**
There were some bits of the new specialist document set up for the UK market conformity bodies that didn't quite work, or were changed after seeing what they actually look like in the wild.

They are:
- We'd overridden the slug which meant that the logic to derive which metadata partial to use broke
- The labels on the form fields weren't quite in the right place which meant we fell back to using the default
- We had a `body name` field defined, but we can use the existing `title` field instead
- A small tweak to put UK at the top of the select drop down
- Added some tests which just make sure that basic stuff works OK

I've disabled some validation (temporarily) as we're about to create and partially import a bunch of things.  We'll re-enable this once the initial tranche is complete.

https://trello.com/c/ayXlI8nK/594-automatically-generate-the-public-bodies-for-ops-to-tag-to-the-nandos-finder

### CODE REVIEW

File: app/models/uk_market_conformity_assessment_body.rb [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
-  validates :uk_market_conformity_assessment_body_legislative_area, presence: true
+  # validates :updated_at, presence: true, date: true
+  # validates :uk_market_conformity_assessment_body_number, presence: true
+  # validates :uk_market_conformity_assessment_body_type, presence: true
+  # validates :uk_market_conformity_assessment_body_registered_office_location, presence: true
+  # validates :uk_market_conformity_assessment_body_testing_locations, presence: true
+  # validates :uk_market_conformity_assessment_body_website, presence: true
+  # validates :uk_market_conformity_assessment_body_email, presence: true
+  # validates :uk_market_conformity_assessment_body_phone, presence: true
+  # validates :uk_market_conformity_assessment_body_legislative_area, presence: true
```
Comment (bilbof) [inline app/models/uk_market_conformity_assessment_body.rb:10]: Could these be deleted? Or do we want to use them in future?

**Review by bilbof — APPROVED** [review]
LGTM

**Review by sophiebrown01 — APPROVED** [review]
Looks good 👍

File: app/models/uk_market_conformity_assessment_body.rb (hunk 1 above)
Comment (sihugh) [reply app/models/uk_market_conformity_assessment_body.rb:10]: They'll be reactivated as soon as we've done the initial import 👍

**Comment by sihugh** [comment]
Thanks both!

**Pull request merged**